cli/cli.go

```diff
@@ -94,6 +94,71 @@ func main() {
 				},
 			},
 		},
+		{
+			Name: "delete",
+			Subcommands: []cli.Command{
+				{
+					Name:      "pipeline",
+					Usage:     "delete pipeline",
+					ArgsUsage: "<pipeline-name>",
+					Before:    requireNameArg,
+					Action:    deletePipeline,
+				},
+				{
+					Name:      "build",
+					Usage:     "delete build",
+					ArgsUsage: "<pipeline-name>",
+					Flags: []cli.Flag{
+						cli.IntFlag{
+							Name:  "build, b",
+							Usage: "build number, if not provided will not proceed deletion",
+						},
+					},
+					Before: requireNameArg,
+					Action: deleteBuild,
+				},
+			},
+		},
+		{
+			Name:  "deploy",
+			Usage: "deploy kontinuous app in the cluster",
+			Subcommands: []cli.Command{
+				{
+					Name:   "remove",
+					Usage:  "remove kontinuous app in the cluster",
+					Action: removeDeployedApp,
+				},
+			},
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "namespace",
+					Usage: "Required, Kubernetes namespace to deploy kontinuous",
+					Value: "kontinuous",
+				},
+				cli.StringFlag{
+					Name:  "s3-access-key",
+					Usage: "Required, S3 access key",
+				},
+				cli.StringFlag{
+					Name:  "s3-secret-key",
+					Usage: "Required, S3 secret key",
+				},
```

Comment:
Planning on removing this: `--s3-access-key` and `--s3-secret-key`. This was added so we can define the access and secret key of minio. For a bootstrap deploy, I don't think the user needs to define these, so we should remove these flags and let minio auto-generate the access and secret keys. 

So with this, we can narrow down the flags to just `--namespace`

the `auth-secret` needs to be inputed like the gh `user`/`pass`/`2fa` as they are sensitive.